src/Compatibility/ApiCompat/Microsoft.DotNet.ApiCompat.Tool/Program.cs

```diff
@@ -19,94 +19,94 @@ static int Main(string[] args)
             // Important: Keep parameters exposed in sync with the msbuild task frontend.
```

Comment:
Funnily, My ApiDiff PR is using this newer version of System.CommandLine, and had to only use it in the new tool but not in the existing ones:

https://github.com/dotnet/sdk/pull/46425

I'll soon merge my PR and then there might (or might not) be a merge conflict in this PR, so letting you know in advance. What we (I can do it) need to update in this PR after I merge mine, is remove the exception for the ApiDiff tool so that it instead uses the same version as everyone else.

Reply:
On a second thought, I don't want to block you. I'll merge my PR _after_ the prev4 snap, and I'll fix my PR on top of yours.

Reply:
@carlossanlop thanks a lot, I really appreciate it!